src/components/pages/Create.vue

```diff
@@ -46,14 +45,37 @@ export default {
   },
   methods: {
     sendGarment() {
-      axios.post('http://localhost:3000/garments/', this.garment)
-        .then((response) => {
-          console.log(response.data);
-        })
-        .catch(error => this.showError(error.message));
-    },
-    showError(error) {
-      EventBus.$emit('showError', error);
+      const gh = new GitHub({
+        token: 'insert your token here',
+      });
+      const options = {
+        name: this.garment.title,
+        description: this.garment.title,
+        homepage: 'http://example.com',
+        auto_init: true,
+        AcceptHeader: 'mercy-preview+json',
+      };
+
+      gh.getUser().getProfile().then(function (profileResponse) {
+        console.log(profileResponse.data.login); // Success!
+        const username = profileResponse.data.login;
+        gh.getUser(username).createRepo(options).then(function (createResponse) {
+          const remoteRepo = gh.getRepo(username, createResponse.data.name);
+          const obj = {
+            title: 'github-for-fashion',
+          };
+          remoteRepo.writeFile('master', 'garment-config.json', JSON.stringify(obj), 'Garment project setup', function () {}).then(function (repoResponse) {
```

Comment:
* Use a variable for `'garment-config.json'`
* what is the empty callback for?
* new line for the `.then`